lms/static/sass/_experiments.scss

```diff
@@ -426,17 +430,25 @@
   .upsell-certificate {
     height: 43px;
     position: absolute;
-    left: 40%;
-    margin-top: 29px;
+    margin: 0;
+    bottom: -58px;
+    left: 220px;
   }
 
   .upsell-modal-checkmark-group {
     list-style: none;
-    margin-left: 0;
+    padding-left: 30px;
+    margin-bottom: 15px;
+
+    li {
+      margin-bottom: 5px;
```

Comment:
I would bump this to ```15px```;

Reply:
probably won't need to do this due to the height my pr is adding, but i'll probably have to tweak margin there anyway